src/ConfigManager.cpp

```diff
@@ -251,18 +243,18 @@ bool ConfigManager::wifiConnected() {
 }
 
 void ConfigManager::setup() {
-    char magic[2];
-    char ssid[32];
-    char password[64];
+    char magic[WIFI_OFFSET];
```

Comment:
With this kind of usage, perhaps this should be renamed, or a new const `MAGIC_LENGTH` should be created?

Reply:
yeah, would make it cleaner

Reply:
yep